website/src/utils/utils.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import { isAddress } from './web3-utils'
+const PROFILE_PREFIX = 'PROFILE_DATA'
+
+export function isProfileEmpty ({ name, description, icon, links }) {
+  return !name && !description && !icon && !links.length
+}
+
+export function composeMessage (address, name, description, icon, links) {
+  const concatenatedLinks = links.reduce((concatenatedLinks, link) => `${concatenatedLinks}${link}`, '')
+  return `${PROFILE_PREFIX}${address}${name.replace(/\s/g, '')}${description.replace(/\s/g, '')}${icon}${concatenatedLinks}`
+}
+
+export function constructPathDescription (transactionPath, proxies) {
+  const preTransactionPath = transactionPath.slice(0, transactionPath.length - 1)
+  const transactionDescription = preTransactionPath.map(({ description, to }, index) => {
+    //
```

Comment:
```suggestion

```

Reply:
Done in https://github.com/1Hive/apiary/pull/181/commits/1adad996efefbaa949de977d8508b6bbc2f7684f.